<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win_new_project_dialogue</class>
 <widget class="QMainWindow" name="win_new_project_dialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Project</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::FolderNew"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_help_new_project">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>160</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DialogQuestion"/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_question_dialog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>531</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start from an existing data base?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_new_elf">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>160</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Load New ELF</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Load_json">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>160</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Load JSON</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
